### PR TITLE
Increase the specificity for footer links

### LINKED ISSUE #2326: Blue text on blue background in footer
Prerequisites:
Build identifier: Mozilla/5.0 (Windows NT 6.1; Win64; x64; rv:45.0) Gecko/20100101 Firefox/45.0

Steps to reproduce:
1.Load https://addons-dev.allizom.org/en-US/firefox/
2.Observe the footer

Expected results:
Text from footer links is white

Actual results:
Blue text on blue background in footer

Please see screencast for this bug : http://screencast.com/t/gcLy27aBXlWq

### PR DESCRIPTION
**Pull request by muffinresearch**
Fixes #2326 

privacy___security____add-ons_for_firefox

### CODE REVIEW

File: static/css/restyle/restyle.less [hunk 1]
```diff
@@ -1049,7 +1049,7 @@ body:not(.developer-hub) section.secondary {
   font-style: normal;
 
   .links-footer a {
-    color: @link-on-color-bg;
+    color: @link-on-color-bg !important;
```
Comment (tofumatt) [inline static/css/restyle/restyle.less]: That's certainly one way! :wink:

**Comment by tofumatt** [comment]
r+

File: static/css/restyle/restyle.less (hunk 1 above)
Comment (muffinresearch) [inline static/css/restyle/restyle.less]: hammertime

**Comment by wagnerand** [comment]
Ugh, yeah, I didn't notice that changing form `id` to `class` broke it.

**Pull request merged**